### PR TITLE
[Day Commit]:

### PR DESCRIPTION
**Pull request by pushpendranirwan**
Kindly review and let me know your feedback.

### CODE REVIEW

File: vfox-restServer/src/main/java/com/vfoexchange/restServer/component/DatabaseComponent.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.stereotype.Component;
+
+import java.util.Objects;
+
+@Component
+public class DatabaseComponent {
+
+    @Value("${jdbc.url}")
+    private String url;
+    @Value("${jdbc.driverClassName}")
```
Comment (ajayvfox) [inline vfox-restServer/src/main/java/com/vfoexchange/restServer/component/DatabaseComponent.java]: Please add blank lines in between variable declarations.

File: vfox-restServer/src/main/java/com/vfoexchange/restServer/component/MailComponent.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MailComponent {
+
+    @Value("${spring.mail.host}")
+    private String hostName;
+    @Value("${spring.mail.port}")
```
Comment (ajayvfox) [inline vfox-restServer/src/main/java/com/vfoexchange/restServer/component/MailComponent.java]: Please add blank lines in between variable declarations.

File: vfox-restServer/src/main/java/com/vfoexchange/restServer/controller/UserController.java [hunk 3]
```diff
@@ -26,16 +28,16 @@
 @RestController
 public class UserController {
     private static Logger LOGGER = LoggerFactory.getLogger(UserController.class);
-    @Autowired
-    private Environment environment;
+
     @Autowired
     private UserService userService;
     @Autowired
```
Comment (ajayvfox) [inline vfox-restServer/src/main/java/com/vfoexchange/restServer/controller/UserController.java]: Please add blank lines in between variable declarations.

File: vfox-restServer/src/main/java/com/vfoexchange/restServer/coreApp/DbConfig.java [hunk 4]
```diff
@@ -24,15 +27,17 @@
 public class DbConfig {
 
     @Autowired
-    private Environment env;
+    private DatabaseComponent database;
+    @Autowired
```
Comment (ajayvfox) [inline vfox-restServer/src/main/java/com/vfoexchange/restServer/coreApp/DbConfig.java]: You missed one -:)

Comment (pushpendranirwan) [reply vfox-restServer/src/main/java/com/vfoexchange/restServer/coreApp/DbConfig.java]: Done

**Review by ajayvfox — APPROVED** [review]
LGTM

**Pull request merged**